### DISCUSSION TITLE
Not editable field still editable

### DISCUSSION THREAD

**Discussion by jdevries14** [pimcore/pimcore#10442] [Q&A / Support]
When making a field not editable, saving it the field is still not editable. (Also when reloading the object it self, or close it and reopen it)

<img width="618" alt="Screenshot 2021-09-28 at 20 23 35" src="https://user-images.githubusercontent.com/44669189/135144116-a5940776-36b7-4ca9-8e4e-9b1e0841440d.png">

As mentioned: 
<img width="409" alt="Screenshot 2021-09-28 at 20 24 37" src="https://user-images.githubusercontent.com/44669189/135144263-a378e642-0a64-47f7-92f6-3721fb0a6b25.png">

**Comment by jdevries14** [comment]
It seems to work on Safari

![Uploading Screenshot 2021-09-28 at 20.29.53.png…]()

**Comment by jdevries14** [comment]
When using Incognito it also works, so probably a cache issue. How would we fix this?

**Comment by dvesh3** [comment]
@jdevries14 This is already fixed for 10.1.4 https://github.com/pimcore/pimcore/issues/10424. Please try this patch https://patch-diff.githubusercontent.com/raw/pimcore/pimcore/pull/10433.patch to confirm. thanks!
